### ISSUE TITLE
Visual Studio 2022 Support

### ISSUE THREAD

**Issue by scottkuhl** [madskristensen/EditorConfigLanguage#114]
### Installed product versions
- Visual Studio: 2022 All Versions

### Description
Make the extension available in Visual Studio 2022.

### Steps to recreate
1. Download the extension installer.
2. Run the installer.

### Current behavior
Message that it is already installed to all applicable products appears even though it is not installed in Visual Studio 2022.
It is also not listed in Online Extensions in Manage Extensions Visual Studio 2022.

### Expected behavior
Installs and works in Visual Studio 2022.

_Note: Sorry for the all the extra details.  I probably could have stopped with the title on this one._

**Comment by julealgon** [comment]
Any updates on this? Installed VS2022 today and went through my extensions. This was one among my list that didn't show up in VS 2022.

**Comment by craigktreasure** [comment]
Personally, i'm not a fan of the new UI for interacting with `.editorconfig` files in Visual Studio 2022. It doesn't show all of the available options (especially in light of .NET 6 and C# 10) and is a bit buggy (sometimes doesn't commit the change correctly to file). I'd prefer to edit the file as before in 2019 in text form enlightened by this extension.

**Comment by julealgon** [comment]
> I'd prefer to edit the file as before in 2019 in text form enlightened by this extension.

Even if editing the file was not the default, the option should be there IMHO. Sometimes the UI is too limiting, as you said, and you want to drop into the actual contents directly. Doing that right now is harder (you have to "try" since it will always show the UI and provide no option to show the contents) and when you do, we get back to lack of editor support.

This should probably be treated as a xaml or forms file, where you have the designer (the UI) _and_ the text editor.

I was expecting this editor here to be native on VS 2022 to be honest.

**Comment by madskristensen** [comment]
Let's update the https://github.com/madskristensen/EditorConfigLanguage/blob/master/src/Schema/EditorConfig.json file with all the latest analyzer rules as well.

**Comment by John0King** [comment]
@craigktreasure 
> Personally, i'm not a fan of the new UI for interacting with .editorconfig files in Visual Studio 2022.

Me neither.

and Let's feedback on here also https://developercommunity.visualstudio.com/t/editconfig-ui-and-editconfig-text-editor-should-be/1561251?from=email&viewtype=all

looking forwoard for this extension to be the buildin one

**Comment by shmuelie** [comment]
@madskristensen, any update on this?

**Comment by madskristensen** [comment]
No, I expect it would conflict with the built-in editorconfig designer in VS 2022

**Comment by giggio** [comment]
We should be able to still use it, if the person installed it they want a text editor, not the default experience.

**Comment by madskristensen** [comment]
Fixed
